bcipy/helpers/convert.py

```diff
@@ -80,12 +124,15 @@ def write_edf(output_path: str,
     # set conversion parameters
     dmin, dmax = [-32768, 32767]
     pmin, pmax = [raw_data.min(), raw_data.max()]
+
     n_channels = len(raw_data)
 
     try:
         writer = EdfWriter(str(output_path),
                            n_channels=n_channels,
                            file_type=FILETYPE_EDFPLUS)
+        if annotation_channels:
+            writer.set_number_of_annotation_signals(annotation_channels)
```

Comment:
I read through the documentation for this method and it seems like this is needed any time the number of triggers/annotations exceeds the number of seconds in the experiment. Wouldn't this almost always be the case since we usually show about 3 to 4 stim per second? Should we try to calculate this from the parameters (using stim_length)?

Reply:
In practice this wasn't clear cut and forcing the annotation channels led to some issues. I added a warning method to let the user know some triggers may be lost and to increment gradually.